lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -288,6 +291,27 @@ def update_sample(db: DatabaseSession,
     return sample
 
 
+def update_details_nwgc_id(sample: Any, additional_details: dict) -> dict:
+    """
+    Given a *sample* fetched from `warehouse.sample`,
+    extend `sample.details.nwgc_id` to an array if needed.
+
+    Add provided "nwgc_id" within *additional_details* to the existing array
+    if it doesn't already exist
+    """
+    exisiting_nwgc_ids = sample.details["nwgc_id"]
+    new_nwgc_ids = additional_details["nwgc_id"]
+
+    # Extend details.nwgc_id to an array
+    if isinstance(sample.details["nwgc_id"], int):
+        exisiting_nwgc_ids = [sample.details["nwgc_id"]]
```

Comment:
I think better here to encapsulate in a list _any_ value that isn't already a list, not just ints specifically, e.g. making the condition `if not isinstance(…, list):`.

For example, if one of the NGWC ids is a string, then a TypeError will be thrown because of the `+` operator below. I realize that's not the case right now, but it could easily happen in the future and good to be defensive.